lib/features/token-simulation-behavior/TokenSimulationBehavior.js

```diff
@@ -38,7 +38,8 @@ function TokenSimulationBehavior(eventBus, animation, injector) {
     'bpmn:ScriptTask',
     'bpmn:ServiceTask',
     'bpmn:Task',
-    'bpmn:UserTask'
+    'bpmn:UserTask',
+    'bpmn:IntermediateThrowEvent' // works same way as a simple task from the token perspective

```

Comment:
It's a good thought to add such a comment to explain certain stuff. However, since the intermediate throw event is handled by the TaskHandler now, it's declarative enough. Let's keep our code base clean and remove the comment.
```suggestion
    'bpmn:IntermediateThrowEvent' 
```